test/remote/gateways/remote_reach_test.rb

```diff
@@ -140,6 +141,48 @@ def test_successful_authorize_and_capture
     assert_success response
   end
 
+  def test_successful_purchase_with_store_credentials
+    @options[:stored_credential] = { initiator: 'cardholder', initial_transaction: true, reason_type: 'installment' }
+    response = @gateway.purchase(@amount, @credit_card, @options)
+
+    assert_success response
+
+    assert response.params['response'][:Authorized]
+    assert response.params['response'][:OrderId]
```

Comment:
This came up in a previous PR as well. The mixed use of strings and symbols indicates to me that something is incorrect between getting a response from the gateway and building the AM `Response` object. I think it was resolved in a previous PR, so it might resolve itself when that is merged in, but just want to call it out just in case.

Reply:
Yes, I think it should be corrected in a separate issue. This is fine for now.